### ISSUE TITLE
DEC Rainbow RX50 disks MS-DOS/FAT file system support

### ISSUE THREAD

**Issue by jfcarrano** [davidgiven/fluxengine#719]
From my testing it looks like [filesystem access](https://cowlark.com/fluxengine/doc/filesystem.html) for DEC Rainbow RX50 disks with MS-DOS/FAT are not supported by fluxengine. Is that correct? If so, would it be possible to add that? I'm hoping to copy some files out of the disk images I have.

**Comment by davidgiven** [comment]
Yes, should be --- it just depends on how quirky the format is. The first thing to try is just adding `filesystem.fatfs=` to the custom configuration (the button to the right of the format dropdown). That'll use the default settings.

**Comment by davidgiven** [comment]
Urgh.

Try `filesystem.fatfs={}`. It's been a while since I touched this stuff...

**Comment by jfcarrano** [comment]
Sadly, getting the same error with that too

**Comment by davidgiven** [comment]
Great, looks like I broke that.

If you create a text file containing:

```
filesystem { fatfs {} }
```

...save it as `fat.textpb` in the current directory, then you should be able to put `fat.textpb` in the extra config and it'll load the settings from that file.

**Comment by jfcarrano** [comment]
The other error is gone but now I am back to: "no filesystem configured" when attempting to browse files. I have the textpb file in the directory with the GUI and am launching it from there. Is that the correct spot for that?

**Comment by davidgiven** [comment]
Sorry for the delay, been doing bugfixes...

Also, I'm afraid I misled you. I changed the way this works and then forgot about it. Try:

filesystem.type=FATFS

...in the extra config box. You don't need any files.

**Comment by jfcarrano** [comment]
I'm curious if you've made any progress on this? No worries if not, just figuring out if I should go attempt some of the other methods I've seen to extract files from these disk images
